crates/server-primitives/src/jsonrpc.rs

```diff
@@ -233,9 +175,8 @@ impl MutateResponse {
 #[error("MutateError")]
 #[serde(tag = "type", content = "data")]
 #[non_exhaustive]
-pub enum MutateError {
+pub enum ExecuteError {
     SerdeError { message: String },
     CallError(CallError),
     FunctionCallError(String),
-}
-// *************************************************************************
+}
```

Comment:
eof

Reply:
wdym? :)

Reply:
```suggestion
}

```
end of file. To match other files in project where all end with space

Reply:
Oh 😂